js/ui/panel.js

```diff
@@ -2675,8 +2675,8 @@ Panel.prototype = {
             if (Main.panelManager && horizontal_panel) {
                 let panels = Main.panelManager.getPanelsInMonitor(this.monitorIndex);
                 for (let p = 0, len = panels.length; p < len; p++) {
-                    if (panels[i].panelPosition == PanelLoc.left || panels[i].panelPosition == PanelLoc.right)
-                        panels[i]._moveResizePanel();
+                    if (panels[p].panelPosition == PanelLoc.left || panels[p].panelPosition == PanelLoc.right)
+                        panels[p]._moveResizePanel();
```

Comment:
yep, that's what I noticed :)  I can remove it from here if needed, but as the changes are identical I imagine there won't be a problem if both are merged.